node-yun-http/node-yun-http.ino

```diff
@@ -321,10 +321,10 @@ void setup()
   FileSystem.begin();
   // a second to initialize:
   delay(1000);
-  // load cell / ADS1231 pin definition: SCL 7, Data 6, PowerDown 10
-  loadCell.attach(7,6,10);
-  // second load cell / ADS1231 pin definition: SCL 8, Data 5, PowerDown 9
-  loadCell01.attach(8,5,9);
+  // load cell / ADS1231 pin definition: Data 6, SCL 7, PowerDown 10
+  loadCell.begin(6,7,10);
+  // second load cell / ADS1231 pin definition: Data 5, SCL 8, PowerDown 9
+  loadCell01.begin(5,8,9);
```

Comment:
At this place, the program will fail to compile, demanding the `pin_SPEED` argument.

```cpp
/Users/amo/dev/hiveeyes/sources/arduino/node-yun-http/node-yun-http.ino: In function 'void setup()':
/Users/amo/dev/hiveeyes/sources/arduino/node-yun-http/node-yun-http.ino:325:24: error: no matching function for call to 'ADS1232::begin(int, int, int)'
   loadCell.begin(6,7,10);
                        ^
In file included from /Users/amo/dev/hiveeyes/sources/arduino/node-yun-http/node-yun-http.ino:98:0:
.pio/libdeps/uno/ADS1232/src/ADS1232.h:45:14: note: candidate: void ADS1232::begin(byte, byte, byte, byte, Speed)
         void begin(byte pin_DOUT, byte pin_SCLK, byte pin_PDWN, byte pin_SPEED, Speed = SLOW);
              ^~~~~
.pio/libdeps/uno/ADS1232/src/ADS1232.h:45:14: note:   candidate expects 5 arguments, 3 provided
```

Reply:
Maybe the ADS1231 just does not have this pin, so it is effectively not compatible with an ADS1232 library at all?